### PR TITLE
Remove whitespaces

### PR DESCRIPTION
**Pull request by shavitush**

### CODE REVIEW

File: scripting/include/smlib/strings.inc [hunk 1]
```diff
@@ -87,15 +87,15 @@ stock String_Trim(const String:str[], String:output[], size, const String:chrs[]
...
+{
 	for (new i=0; i < size; i++) {
 		ReplaceString(buffer, SIZE_OF_INT, removeList[i], "", caseSensitive);
 	}
 }
 
 /**
  * Converts the whole String to lower case.
- * Only works with alphabetical characters (not ���) because Sourcemod suxx !
+ * Only works with alphabetical characters (not ���) because Sourcemod suxx !
```
Comment (peace-maker) [inline scripting/include/smlib/strings.inc]: Looks like your editor messed up the file encoding.

File: scripting/include/smlib/strings.inc [hunk 2]
```diff
@@ -120,7 +120,7 @@ stock String_ToLower(const String:input[], String:output[], size)
 
 /**
  * Converts the whole String to upper case.
- * Only works with alphabetical characters (not ���) because Sourcemod suxx !
+ * Only works with alphabetical characters (not ���) because Sourcemod suxx !
```
Comment (peace-maker) [inline scripting/include/smlib/strings.inc]: Same

**Comment by shavitush** [comment]
I sort of insist on removing whitespaces so I could actually contribute more without changing my development environment, Atom removes those automatically and I prefer it like that. I'll fix the encoding

**Comment by peace-maker** [comment]
Can you squash those two commits into one? You should create a new branch for each PR you want to do in your own fork, so you can keep doing other stuff while waiting for one PR to be reviewed!

**Comment by shavitush** [comment]
Git is confusing honestly, I think this is what you wanted?

**Comment by peace-maker** [comment]
Sorry, no, squashing the last - now - 3 commits into one. So the commit history isn't polluted with random fixup commits from PRs.

**Comment by shavitush** [comment]
Alright @peace-maker - I understood how to squash :)

**Comment by peace-maker** [comment]
Thank you

**Pull request merged**